src/hooks/CacheProvider.tsx

```diff
@@ -0,0 +1,42 @@
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import React, { createContext, useContext } from 'react';
+import { Cache } from 'react-native-cache';
+
+type CacheContextProps = {
+  cache?: Cache;
+};
+
+export const CacheContext = createContext<CacheContextProps>({
+  cache: undefined,
+});
+
+export const CacheContextProvider = ({
+  children,
+}: {
+  children?: React.ReactNode;
+}) => {
+  return (
+    <CacheContext.Provider
+      value={{
+        cache: new Cache({
+          namespace: 'LO-SDK',
+          policy: {
+            maxEntries: 5000, // if unspecified, it can have unlimited entries
+            stdTTL: 60 * 60 * 24, // the standard ttl as number in seconds, default: 0 (unlimited)
+          },
+          backend: AsyncStorage,
```

Comment:
I purpose we standardize on using this caching mechanism where ever local caching is required (thinking appConfig).